src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.spec.ts

```diff
@@ -79,6 +82,14 @@ describe('DotEditPageWorkflowsActionsComponent', () => {
             expect(actionsButton.model[2].label).toEqual('Save / Publish');
         });
 
+        it('should get workflow actions when pageState changes"', () => {
+            component.pageState = {
+                workingInode: 'cc2cdf9c-a20d-4862-9454-2a76c1132123b'
+            };
+            component.ngOnChanges();
+            expect(dotWorkflowService.getContentWorkflowActions).toHaveBeenCalledTimes(2);
```

Comment:
also you have to check that this is called with the right parameter.

Reply:
done